src/storage/mod.rs

```diff
@@ -481,6 +486,8 @@ impl Storage {
             return Err(box_err!("failed to join sched_handle, err:{:?}", e));
         }
 
+        self.gc_worker.stop()?;
```

Comment:
Should join?

Reply:
`GCWorker`'s `stop` method has joined the worker itself.